package.json

```diff
@@ -37,16 +47,29 @@
     "winston": "^2.3.1"
   },
   "devDependencies": {
```

Comment:
NOTE: I did add some dependencies here. They're all dev-only, but good to review.